addons/website/static/src/js/menu/edit.js

```diff
@@ -116,7 +116,9 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
             self.destroy();
         };
         if (!this.wysiwyg.isDirty()) {
-            return destroy();
+            destroy();
+            window.location.reload();
+            return;
```

Comment:
I'm wondering here as well. What will this do when the website is in an iframe in the webclient ?

Reply:
For the iframe I have no idea. probably will need a change indeed. 
But it's not yet merged, and will only be in master right ?

I've added the `windows.location.reload()` to reflect what is append at the end of the `saveToServer()` https://github.com/odoo/odoo/blob/15.0/addons/web_editor/static/src/js/wysiwyg/wysiwyg.js#L819